account_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -81,6 +85,7 @@ class OnlineBankStatementProvider(models.Model):
             ("monthly", "Monthly statements"),
         ],
         default="daily",
+        string="Transactions synchronization",
```

Comment:
Sorry again to change the text, but thinking twice, this is not the frequency of downloading, so this label will confuse users. It's more the packing option when downloading. If you put monthly, on next synchronization, it will download the whole current month. Which label do you think it should put?

Reply:
"Statement Grouping" sounds like a good label. What do you think? ;)

Reply:
Uhm, not exactly, as there's no statement with the new option, and it's more a question of "what to download". Also the labels of each option are not correct (but we can keep the values for not doing migration scripts). Thinking in something...

Reply:
It's a bit long, but "Transactions interval to obtain" is defining exactly what is doing, and the labels should be changed to "Day", "Week", "Month" instead of the adverbs.

Reply:
@pedrobaeza I’ve updated the code. Is it Ok for you?